test/docker-e2e/e2e_test.go

```diff
@@ -41,6 +44,61 @@ func (s *CelestiaTestSuite) SetupSuite() {
 	s.client, s.network = tastoradockertypes.DockerSetup(s.T())
 }
 
+// ExecuteNodeCommand executes a command on a chain node with common parameters automatically added.
+// This reduces boilerplate in tests by setting common flags like --home, --node, --fees, etc.
+func (s *CelestiaTestSuite) ExecuteNodeCommand(ctx context.Context, chainNode tastoratypes.ChainNode, cmd ...string) (string, string, error) {
```

Comment:
see comments below

this logic is sloppy imo and should be refactored.

Reply:
This is now refactored to use a builder pattern